as botan is already packaged in debian we don't want to build it from source

diff --git a/Cargo.toml b/Cargo.toml
index e74450f..35d2f7f 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -30,15 +30,15 @@ license = "MIT"
 repository = "https://github.com/randombit/botan-rs"
 resolver = "1"
 
-[build-dependencies.botan-src]
-version = "0.30101.2"
-optional = true
+#[build-dependencies.botan-src]
+#version = "0.30101.2"
+#optional = true
 
 [features]
 botan3 = []
 default = []
 no-std = []
-vendored = [
-    "botan-src",
-    "botan3",
-]
+#vendored = [
+#    "botan-src",
+#    "botan3",
+#]
